makefiles/pseudomodules.inc.mk

```diff
@@ -71,10 +78,13 @@ PSEUDOMODULES += gcoap_fileserver
 PSEUDOMODULES += gcoap_fileserver_delete
 PSEUDOMODULES += gcoap_fileserver_put
 PSEUDOMODULES += gcoap_dtls
-## Enable @ref net_gcoap_dns
+## @addtogroup net_gcoap_dns
+## @{
+## Enable the @ref gcoap_dns function
 PSEUDOMODULES += gcoap_dns
 ## Enable the @ref gcoap_dns_server_proxy_set function
 PSEUDOMODULES += gcoap_dns_proxied
+## @}
```

Comment:
Those lines were added, because otherwise Doxygen would include these to `gnrc_netdev_default`'s documentation.